docs/handbook/getting-started.md

```diff
@@ -1933,12 +1949,17 @@ The beadm command impacts the non-global zones in your boot environments as foll
 
 ## The X-Window system
 
-Nearly all cards can use the VESA driver, and are therefore supported for 2D.
+### Video card support (2D)
+
+* Nearly all cards can use the VESA driver, and are therefore supported for 2D.
+* 2D acceleration for NVidia GPU's is provided by the Nvidia binary driver.
+* Work has begun to provide Radeon (2D) acceleration support for both legacy and modern AMD GPU's.
```

Comment:
@pyhalov, Well now I am confused. What has been here since OpenSolaris? 2D support for Radeon? 
If so, then what is Ken is now working on restoring must be 2D support which was dropped due the version bump of Xorg?